src/viewer.js

```diff
@@ -121,12 +110,69 @@ function createViewer(container, data) {
   const renderer = fullScreenRenderer.getRenderer();
   const renderWindow = fullScreenRenderer.getRenderWindow();
 
+  const dataArray = data.image.getPointData().getScalars();
+  if (!dataArray) {
+    window.alert('No data array available in dataset');
+  }
+  const dataRange = dataArray.getRange();
+
+  const lookupTable = vtkColorTransferFunction.newInstance();
+  // TODO: Make colormap selectable
+  lookupTable.applyColorMap(vtkColorMaps[2]);
+  lookupTable.setMappingRange(...dataRange);
+  lookupTable.updateRange();
+
+  const transferFunctionWidget = vtkPiecewiseGaussianWidget.newInstance({ numberOfBins: 256, size: [400, 150] });
+  transferFunctionWidget.updateStyle({
+    backgroundColor: 'rgba(255, 255, 255, 0.6)',
+    histogramColor: 'rgba(100, 100, 100, 0.5)',
+    strokeColor: 'rgb(0, 0, 0)',
+    activeColor: 'rgb(255, 255, 255)',
+    handleColor: 'rgb(50, 150, 50)',
+    buttonDisableFillColor: 'rgba(255, 255, 255, 0.5)',
+    buttonDisableStrokeColor: 'rgba(0, 0, 0, 0.5)',
+    buttonStrokeColor: 'rgba(0, 0, 0, 1)',
+    buttonFillColor: 'rgba(255, 255, 255, 1)',
+    strokeWidth: 2,
+    activeStrokeWidth: 3,
+    buttonStrokeWidth: 1.5,
+    handleWidth: 3,
+    iconSize: 20, // Can be 0 if you want to remove buttons (dblClick for (+) / rightClick for (-))
```

Comment:
The buttons are kind of optional if you know (+) double click (-) right click.

Reply:
I think it is nice to have them explicitly there -- hard to guess that behavior otherwise.

Reply:
ok no worries.. Without them you can have slicker UI.